Cargo.toml

```diff
@@ -35,9 +35,10 @@ dashmap = "5.5.0"
 text-diff = "0.4.0"
 tower-lsp = "0.19.0"
 tokio = { version = "1.29.1", features = ["full"] }
-
-[patch.crates-io]
-tree-sitter = { git = "https://github.com/tree-sitter/tree-sitter", rev = "c16b90d" }
+better_any = "0.2.0"
 
 [[bin]]
 name = "tree-sitter-lint"
+
+[patch.crates-io]
```

Comment:
I should move this back up I liked it next to the `[dependencies]` (that it's overriding)

The reason I moved it is that doing eg `cargo remove better_any` deleted it (which is clearly a bug, I should file an issue)

Reply:
Ah nice, is this that thing I was trying to remember the other day where you can override dependencies globally?

Reply:
Yup the one surprising thing I ran into with it is that I need to apply this "patch" on _every_ crate (vs just apply it on `tree-sitter-grep` and then have other crates use its re-exported `tree_sitter_grep::tree_sitter` and that would use the version that `tree-sitter-grep` "insisted on")